modular_zubbers/code/modules/satisfaction/satisfaction.dm

```diff
@@ -0,0 +1,95 @@
+GLOBAL_LIST_EMPTY(needs_chaos)
+GLOBAL_LIST_EMPTY(needs_less_chaos)
+GLOBAL_LIST_EMPTY(needs_antags)
+GLOBAL_LIST_EMPTY(needs_less_antags)
+GLOBAL_LIST_EMPTY(needs_emergency)
+GLOBAL_LIST_EMPTY(needs_less_emergency)
+
+SUBSYSTEM_DEF(round_survey)
+	name = "Round Survey"
+	wait = 30 SECONDS
+	flags = SS_BACKGROUND
+
+/datum/controller/subsystem/round_survey/fire()
+	var/message_string = "Current Round Survey Results : \n"
+	var/list/results = list()
+	for(var/i in GLOB.needs_chaos)
+		results["more chaos"]++
+	if(results["more chaos"])
```

Comment:
"more chaos" etc are prone to being misspelt. These lists are short so I'd honestly just replace invocations of it with just grabbing the list's length